src/adapter/etl-adapter-csv/tests/Flow/ETL/Adapter/CSV/Tests/Integration/CSVExtractorTest.php

```diff
@@ -220,4 +220,46 @@ public function test_loading_data_from_all_partitions() : void
                 ->toArray()
         );
     }
+
+    public function test_extracting_csv_with_more_than_1000_characters_per_line() : void
+    {
+        $extractor = CSV::from(
+            __DIR__ . '/../Fixtures/more_than_1000_characters_per_line.csv'
+        );
+
+        $total = 0;
+        /** @var Rows $rows */
+        foreach ($extractor->extract(new FlowContext(Config::default())) as $rows) {
+            $rows->each(function (Row $row) : void {
+                $this->assertInstanceOf(Row\Entry\ArrayEntry::class, $row->get('row'));
```

Comment:
That is a bit redundant, what I would do here is `->rows(Transform::array_unpack('row'))->drop('row')` and then I would use probably `fetch() : Rows`
Returning ArrayEntry in each row is already tested by other tests, same as extracted values. I would focus this test only on breaking lines